### ISSUE TITLE
Not possible to use style to only double quote keys

### ISSUE THREAD

**Issue by unmarshall** [mikefarah/yq#1677]
**yq version info**

yq (https://github.com/mikefarah/yq/) version 4.25.2
Operating system: mac
Installed via: homebrew

**Describe the bug**

I have a yaml which has a map with quoted keys. I would like to add new key-value pairs to the yaml with double quoted keys.
I went through https://mikefarah.gitbook.io/yq/operators/style#set-double-quote-style but it either offers to double quote only values or both keys and values. In the yaml that i am operating they keys are double quoted but the values are not.

**Input Yaml**
Concise yaml document(s) (as simple as possible to show the bug, please keep it to 10 lines or less)
data1.yml:
```yaml
features:
  "f1": true
  "f2": true
```
What i want to add is another key: "f3" to the map with value true (without any quotes). Is that possible?

**Comment by mikefarah** [comment]
Yep - you can use the "key" operator to work with the key of a map. To set the style of the key you can:


```
.features.f3 | key style |= "double"
```

This can be put together with assigning the value `true` using `with`

```
'with(.features.f3; . = true | key style= "double")'
```

- `with` changed the context to `.feaures.f3`
- `. = true`, the `.` refers to the current node, which is the value
- `key style= "double", `key` will get the key node of the value, `style=` will update the style

Hope that makes sense!
